spec/installation_spec.sh

```
@@ -56,6 +56,38 @@ fDescribe 'bin/install.e-bash.sh'
+
+    # Set up git remote and branches as the install script would
+    git remote add e-bash https://github.com/OleksandrKucherenko/e-bash.git 2>/dev/null || true
+    git branch e-bash-scripts 2>/dev/null || true
+
+    # Return success
+    return 0
+  }
+
+  # Mock upgrade state - creates previous version file
+  mock_upgrade() {
+    local previous_version="${1:-v1.0.0}"
```

Shellcheck (0.11.0)

[warning] previous_version appears unused. Verify use (or export if used externally).

(SC2034)